src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -329,9 +329,12 @@ private function getSchema(): Schema
         return $schema;
     }
 
-    public static function formatDateTime(\DateTimeInterface $dateTime)
+    /**
+     * @internal
+     */
+    public function formatDateTime(\DateTimeInterface $dateTime)
```

Comment:
Does it need to be public? I didn’t spot the usage.

Reply:
This is only use in the `Connection` class so we can make it `private`.

There is a discussion with @Tobion and @nicolas-grekas about the fact that this would introduce a bc break.

IMHO this component is still marked as experimental so we can introduce this bc break. (there were some breaks with the 4.2 -> 4.3 update)